src/rgw/rgw_aio_throttle.cc

```diff
@@ -146,7 +146,7 @@ AioResultList YieldingAioThrottle::get(const RGWSI_RADOS::Obj& obj,
 
     // register the pending write and initiate the operation
     pending.push_back(*p);
-    std::move(f)(this, *static_cast<AioResult*>(p.get()));
+    f(this, *static_cast<AioResult*>(p.get()));
```

Comment:
The rvalues are supposed to be this way, indicating that the functions are to be called once and should be called in move context.

Is this to fix a compiler warning or something?